packages/core/tests/proofs.test.ts

```diff
@@ -369,4 +372,288 @@ describe('Present Proof', () => {
       `The proof request contains an attribute group name that matches a predicate group name: age`
     )
   })
+
+  test('attribute names match, same cred def filter', async () => {
+    const attributes = {
+      name: new ProofAttributeInfo({
+        name: 'age',
+        restrictions: [
+          new AttributeFilter({
+            credentialDefinitionId: credDefId,
+          }),
+        ],
+      }),
+      age: new ProofAttributeInfo({
+        name: 'age',
+        restrictions: [
+          new AttributeFilter({
+            credentialDefinitionId: credDefId,
+          }),
+        ],
+      }),
+    }
+
+    await expect(
```

Comment:
I dont think `expect` has to be async. I think the function called inside the expect has to be marked `void` though.

Reply:
@TimoGlastra you told me some times tests fail because the expect is not awaited, right?

Reply:
Ah I just looked at the jest docs and this is definitely how we are supposed to do it. We did not mark this like this in some other tests. Will fix that in a different PR.